openapi/kas-fleet-manager.yaml

```
@@ -1418,6 +1420,7 @@ components:
         updated_at: "2020-10-05T12:56:36.362208Z"
         version: "2.6.0"
         instance_type: standard
+        browser_url: "http://localhost:8080/"
```

When changing this, let's add in the full url since it's an example of the response.  

```suggestion
        browser_url: "https://console.redhat.com/application-services/streams/kafkas/1isy6rq3jki8q0otmjqfd3ocfrg/dashboard"
```